### DISCUSSION TITLE
How to use the binary?

### DISCUSSION THREAD

**Discussion by obones** [mechano/Kingroon_KP3S_Pro_Marlin#2] [General]
Hello,

Thanks for the effort in building this, but because I'm new on all this, it's not all that clear what I should do.
I have a KP3S Pro S1 with stock firmware, what does this bring to the table?
Is there a way to move to this firmware and then go back to the original one?

**Comment by mechano** [comment]
You have the same printer of mine.
I think you don't have BL Touch.

What you have here are two modern technologies to go fast keeping quality like Linear Advance and Input Shaping. But they need loto of study and tuning to have good results.

The ability to do PID autotune both for bed and hotend directlly from menu.

If you don't have BL Touch there's Manual Mesh creation with 4x4 (16) points. It virtually make useless 3D Touch on a so small build area.

A modern, complete and professional UI and lot of functions. 

If you decide to give a try you'll not repent. And if you need a stock firmware you can download one from here:

https://mega.nz/file/1YJiQLhK#b_0ScGrjiGADSlLgUGqHZ7KvEZlHpod7LV9xskmFgdw

**Reply by obones** [reply]
Ah so, if I read you correctly, I can do all this from the LCD touch panel without any need to connect to the computer ?

Nice, I'll give it a try
